recipes/future-fstrings/meta.yaml

```diff
@@ -18,8 +18,10 @@ requirements:
   build:
     - python
     - setuptools
+    - tokenize-rt
   run:
     - python
+    - tokenize-rt
```

Comment:
does conda have a concept of conditional dependencies? `tokenize-rt` is only needed for python<3.6

Reply:
You can do this with a selector (`- tokenize-rt  # [py<36]` ), but that means you can't build this with `python: noarch`